### PR TITLE
mobile-quality 1.0.0

### PR DESCRIPTION
**Pull request by imranMnts**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/mobile-quality/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    summary: Do you want check iOS part?
+    title: Check iOS
+    value_options:
+    - "yes"
+    - "no"
+- config_file_path: null
+  opts:
+    description: |
+      The config file where we have some informations like the current android app size .eg, to compare with next builds and alert if needed.
+      If you want use Environment Variables instead of the config file, do not set this key and add variables. (See README to see all needed variables)
```
Comment (koral--) [inline steps/mobile-quality/1.0.0/step.yml:71]: README should be a clickable link.

File: steps/mobile-quality/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    is_required: true
+    summary: Do you want check iOS part?
+    title: Check iOS
+    value_options:
+    - "yes"
+    - "no"
+- config_file_path: null
+  opts:
+    description: |
+      The config file where we have some informations like the current android app size .eg, to compare with next builds and alert if needed.
```
Comment (koral--) [inline steps/mobile-quality/1.0.0/step.yml:70]:
```suggestion
      The config file where we have some informations like the current android app size eg. to compare with next builds and alert if needed.
```

File: steps/mobile-quality/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    is_required: true
+    summary: Threshold to alert when app size increase (MB)
+    title: Alert threshold
+    value_options:
+    - "3"
+    - "5"
+    - "10"
+    - "20"
+outputs:
+- IOS_PERMISSIONS_COUNT: null
```
Comment (koral--) [inline steps/mobile-quality/1.0.0/step.yml:170]: PERMISSIONS -> PERMISSION

File: steps/mobile-quality/1.0.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+- ionic
+- xamarin
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/mobile-quality/1.0.0/step.yml:38]: Dependency management looks very messy.
You are declaring git which does not seem to be used.
You are using brew inside a script: https://github.com/imranMnts/bitrise-step-mobile-quality/blob/master/step.sh#L29 which is forbidden by the guidelines.

It seems that you haven't read the Step Development Guideline with understanding.

Comment (imranMnts) [reply steps/mobile-quality/1.0.0/step.yml:38]: Ah sorry, looks like I misunderstood this part yes, I fixed it now

**Comment by imranMnts** [comment]
Continues in #2856

**Pull request closed without merging**